src/mapOfPeople.js

```diff
@@ -40,7 +40,14 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const peopleAtTheRoom = new Map();
+  people.forEach((item, seconds) => {
+    if (!peopleAtTheRoom.delete(item)) {
+      peopleAtTheRoom.set(item, seconds);
+      seconds++;
```

Comment:
1. do not mutate input arguments
2. this line does anything, because it is not for loop, anyway forEach call callback with incremented index next time. You do not need to do it manually